recipes/typing-validation/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "typing-validation" %}
+{% set version = "1.2.11.post4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/typing_validation-{{ version }}.tar.gz
+  sha256: 7aed04ecfbda07e63b7266f90e5d096f96344f7facfe04bb081b21e4a9781670
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python
+    - typing_extensions >=4.6.0  # [py<311]
```

Make this a noarch packages as with the other ones above. This will be simpler to maintain
```suggestion
    - typing_extensions >=4.6.0
```